metron-stellar/stellar-common/src/main/scripts/stellar

```diff
@@ -28,7 +28,8 @@ elif [ -e /usr/lib/bigtop-utils/bigtop-detect-javahome ]; then
   . /usr/lib/bigtop-utils/bigtop-detect-javahome
 fi
 
-export HBASE_CONFIGS=/etc/hbase/conf
+export HBASE_CONFIGS=$(hbase classpath)
 export METRON_VERSION=${project.version}
 export METRON_HOME=/usr/metron/$METRON_VERSION
-java $JVMFLAGS -cp "$HBASE_CONFIGS:$METRON_HOME/lib/*" org.apache.metron.stellar.stellar.shell.StellarShell "$@"
+export STELLAR_LIB=$(find $METRON_HOME/lib/ -name metron-enrichment*.jar)
```

Comment:
This will not pick-up any of the Profiler or Management functions, right?

Reply:
We might need to create a JAR specifically tailored for the REPL.

Reply:
So we are just picking the enrichment jar?

Reply:
It will pick up the profiler functions, they're included in the enrichment jar (that's actually what motivated this, `PROFILE_GET` didn't work randomly based on the order of the classpath).

We probably should include the management functions directly, though.  I'll adjust.

Reply:
The current commit should support both profiler and the management function at this point.